src/main/java/org/opentripplanner/routing/api/response/RoutingError.java

```diff
@@ -0,0 +1,13 @@
+package org.opentripplanner.routing.api.response;
+
+public class RoutingError {
+  public final RoutingErrorCode code;
+  public final String message;
+  public final String paramValue;
```

Comment:
The parameter value here is probably not a good idea, I know I suggested it, but since we want typed parameters as input, the "original" string is not available. So the best we can do is to report back witch input field that failed to validate. We should use an enum for the input fields.

Reply:
I have changed it to an enum